docs/user-guide/integrations/task-runner.md

```diff
@@ -9,3 +9,4 @@ Task runner integrations built and maintained by HTMLHint organization and the c
 - [grunt-htmlhint](https://github.com/htmlhint/grunt-htmlhint) - Grunt plugin for HTMLHint.
 - [gulp-htmlhint-inline](https://github.com/htmlhint/gulp-htmlhint-inline) - Gulp plugin for linting inline html.
 - [htmlhint-loader](https://github.com/htmlhint/htmlhint-loader) - webpack loader for HTMLHint.
+- [super-linter](https://github.com/github/super-linter/) - GitHub action - a combination of linters, including HTMLHint.
```

Comment:
Just a little something, but can you please remove the `/` after `Super-Linter` ^^?

Reply:
:-) done